File: Disertatie_backend/DTO/Books/BookCard.cs
using MongoDB.Bson;

namespace Disertatie_backend.DTO.Books
{
    public class BookCard
    {
        public string Title { get; set; }
        public ObjectId Id { get; set; }
        public string Authors { get; set; }
        public double? AverageRating { get; set; }
        public int? RatingsCount { get; set; }
        public int? TextReviewsCount { get; set; }
        public string PublicationDate { get; set; }
        public string Publisher { get; set; }
        public string CoverUrl { get; set; }
    }
}

File: ClientApp/src/app/_models/bookCard.ts
import { ObjectId } from "model/objectId";

export interface BookCard{
    id: ObjectId;
    title: string;
    authors: string;
    averageRating: number;
    ratingCount: number;
    textReviewsCount: number;
    publicationDate: string;
    publisher: string;
    coverUrl: string;
}